_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -1,5 +1,8 @@
 <?php
 include_once( 'class.jetpack-admin-page.php' );
+// Load API endpoint base classes and endpoints for getting the module list fed into the JS Admin Page
+require_once JETPACK__PLUGIN_DIR . '_inc/lib/core-api/class.jetpack-core-api-xmlrpc-consumer-endpoint.php';
+require_once JETPACK__PLUGIN_DIR . '_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php';
```

Comment:
Can you move these lines to right before where the class will be actually used? We shouldn't always load them because the page might not load or might not need this class as pointed by
```php
return; // No need to handle the fallback redirection if we are not on the Jetpack page
```
or
```php
return; // No need for scripts on a fallback page
```

Reply:
Niceee. Thanks @eliorivero !. Addressed in [64a6784].(https://github.com/Automattic/jetpack/pull/7951/commits/64a678491883297a100f904b4c4c0cad8a84d34f)